.github/workflows/canary-integration-test.yml

```diff
@@ -908,3 +908,48 @@ jobs:
       if: failure() && github.event_name == 'pull_request'
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 60
+
+  multus-cluster-network:
```

Comment:
The interesting part of this test will be when we can test a volume with multus. After csi 3.6.1 is integrated we can add that test?